formatTest/unit_tests/expected_output/wrappingTest.re

```diff
@@ -2060,11 +2063,18 @@ type myRecordWithReallyLongName = {
 
 type doubleEqualsRecord =
   myRecordWithReallyLongName =
-    {xx: int, yy: int};
+    {
+      xx: int,
+      yy: int
+    };
```

Comment:
```
type doubleEqualsRecord =
    myRecordWithReallyLongName = {
      xx: int,
      yy: int
    };
```

I think we should somehow lift the left "{" to the line above it so it could be more consistent. Not sure if our current formatting API has the power to do that though.